server/src/test/java/org/elasticsearch/index/shard/IndexShardIT.java

```diff
@@ -416,7 +416,7 @@ public void testStressMaybeFlushOrRollTranslogGeneration() throws Exception {
         final Settings settings;
         if (flush) {
             // size of the operation plus two generations of overhead.
```

Comment:
I am guessing that the change in the next line is due to only keeping one generation now? In that case, this comment needs updating I think.

Reply:
fixed in https://github.com/elastic/elasticsearch/pull/51905/commits/8c91dcfe43d55bef8c109885527cc7a3584f74a4.